http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java

```diff
@@ -106,7 +108,9 @@ public void testHttp2_Internal_H2C() throws Exception {
     await().atMost(1, TimeUnit.MINUTES).untilAtomic(response, is(notNullValue()));
 
     assertThat(response.get())
-      .startsWith("Aloha HTTP_2");
+      .contains("version = HTTP_2")
+      .contains("Aloha from vert.x!")
+      .contains("HTTP_1_1");
```

Comment:
how come it does contain HTTP_1_1 and HTTP_2 ?

Reply:
HTTP_1_1 is used to communicate with `EdgeVerticle`, which then communicates with `HelloHttp2Verticle` using HTTP_2. The response body from `HelloHttp2Verticle` contains something like
```html
<html>
  <body>
    <h1>Aloha from vert.x!</h1>
    <p>version = HTTP_2</p>
  </body>
</html>
```
This response is received by `EdgeVerticle`, which then appends the http version of the request from `Http2IT`, in our case `HTTP_1_1`. So I'm testing that the response contains both.